src/Subscriptions/Broadcasters/PusherBroadcaster.php

```diff
@@ -62,13 +61,11 @@ public function unauthorized(Request $request): JsonResponse
     public function hook(Request $request): JsonResponse
     {
         (new Collection($request->input('events', [])))
-            ->filter(function ($event): bool {
-                return Arr::get($event, 'name') === 'channel_vacated';
+            ->filter(function (array $event): bool {
+                return $event['name'] === 'channel_vacated';
```

Comment:
```suggestion
                return isset($event['name']) && $event['name'] === 'channel_vacated';
```

I would still add a guard just in case Pusher (or someone testing) sends an invalid payload "breaking" the code (warnings/notices are converted to fatal errors in Laaravel).

Reply:
In case of an invalid payload, i would rather crash then to silently fail.